jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,456 @@
+    "file_infos = await metadata_api.get_multiple_file_info(tdmreader.FileList(file_ids))\n",
+    "file_info = file_infos[0]\n",
+    "\n",
+    "test_file_name = get_property(file_info, \"name\")\n",
+    "program_name = get_property(file_info, \"Test~Procedure\")\n",
+    "\n",
+    "group_names = []\n",
+    "channels = []\n",
+    "formatted_properties = []\n",
+    "\n",
+    "for group in file_info.groups:\n",
+    "    group_names.append(group.name)\n",
+    "    channels.append(group.channels)\n",
+    "    max_values_of_group = []    \n",
+    "    min_values_of_group = []    \n",
+    "    mean_values_of_group = []    \n",
+    "\n",
+    "    for channel in group.channels:\n",
+    "        minimum = float(get_property(channel, \"minimum\") or \"NaN\")\n",
+    "        maximum = float(get_property(channel, \"maximum\") or \"NaN\")\n",
+    "        mean_values_of_group.append((minimum + maximum) / 2)\n",
+    "        max_values_of_group.append(maximum)\n",
+    "        min_values_of_group.append(minimum)\n",
+    "\n",
+    "        \n",
+    "    # Calculate statistic values from metadata\n",
+    "    abs_min = max(max_values_of_group)\n",
+    "    abs_max = min(max_values_of_group)\n",
+    "    abs_mean = sum(mean_values_of_group) / float(len(mean_values_of_group))\n",
+    "    formatted_properties.append(f\"Absolute Maximum: {abs_max:.3f} °C\"+\n",
+    "                                f\",Absolute Minimum: {abs_min:.3f} °C\"+\n",
+    "                                f\",Mean Value: {abs_mean:.3f} °C\")\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define the parameters for the notebook.\n",
+    "\n",
+    "a) In a code cell (*called __parameters cell__*), define the parameters. Fill in the needed values/content parameters in the code cell below. E.g.\n",
+    "\n",
+    "   **Defined parameters:**\n",
+    "   - `comment_group_1`: Writes a comment into the box of the first group.<br> \n",
+    "   (Default value = `Checked`)\n",
+    "   - `shown_channel_index`: Any valid channel index of the second group. This channel is plotted in the second graph. <br>\n",
+    "   (Default value = `2`)\n",
+    "\n",
+    "   Your code may look like the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "comment_grp_1": "Checked",
+      "show_channel_of_group": 2
+     }
+    },
+    "tags": [
+     "Parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "comment_group_1 = \"Checked\"\n",
+    "shown_channel_index = 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "b) Select this code cell (*__parameters cell__*) and open on the right sidebar the __Property Inspector__ panel and add the parameters, their default values, to the __Cell Metadata__ code block. For example, your code may look like the following:\n",
```

Comment:
and open the _Property Inspector_ panel on the right sidebar to add the parameters....